src/c++/libos/lwip/lwip_queue.cc

```diff
@@ -1502,8 +1591,10 @@ int dmtr::lwip_queue::rte_eth_tx_burst(size_t &count_out, uint16_t port_id, uint
     DMTR_TRUE(ERANGE, ::rte_eth_dev_is_valid_port(port_id));
     DMTR_NOTNULL(EINVAL, tx_pkts);
 
-    size_t count = ::rte_eth_tx_burst(port_id, queue_id, tx_pkts, nb_pkts);
+    // If count_out packets were sent on the previous burst, they don't have to be sent again
+    size_t count = ::rte_eth_tx_burst(port_id, queue_id, &tx_pkts[count_out], nb_pkts - count_out);
```

Comment:
I think we should handle partial writes outside of this function, i.e. update nb_pkts according to the sent count.